### PR TITLE
Pldevelop

### PR DESCRIPTION
**Pull request by adwojak**
### What does this changes

PESEL and automotive plates for Poland with tests.

### CODE REVIEW

**Comment by adwojak** [comment]
Can You help me fix issues in my pull request? AppVeroy build fails but I think that's not from my fault. Also coverage has decreased. Do I need to write more tests for my functionality to increase tests coverage? Honestly, it is my first contribution to someone's repo and I don't know how to handle that problems :)

File: faker/providers/automotive/pl_PL/__init__.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# coding=utf-8
+
+from __future__ import unicode_literals
+from .. import Provider as AutomotiveProvider
+
+
+def license_formats_regex():
```
Comment (fcurella) [inline faker/providers/automotive/pl_PL/__init__.py]: I think this is unused

**Comment by fcurella** [comment]
Thank you @adwojak !

I've created #955 to fix the build. Once that's merged in, feel free to rebase your branch against master and force-push your branch.

I wouldn't worry about a 0.04 decrease in

File: faker/providers/automotive/pl_PL/__init__.py (hunk 1 above)
Comment (adwojak) [reply faker/providers/automotive/pl_PL/__init__.py]: Done

**Comment by adwojak** [comment]
Thanks for help, all checks are successfull :) So for now, I don't have to do nothing more? :)

**Comment by fcurella** [comment]
@adwojak That's all! Thank you very much :) ✨

**Pull request merged**